### ISSUE TITLE
Instructor Feedback Week 5 - Project Checkpoint 2

### ISSUE THREAD

**Issue by pawaitemadisoncollege** [ocollins/TennisUniverse#2]
Good job getting jsps built out already and working on that aws deploy!

- [ ] A note about issues/feedback. Please add an "X" to check off the checkboxes to indicate you have completed the work or responded to the questions. When all done, then you can close the issue. See example below:
- [x] A completed task.
- [x] Something to keep in mind in general - if there were an error in a Dao, what would the user experience be? You might consider throwing an exception (you can create your own exceptions) that the controller would catch and then route the user to a friendly-looking error page.
- [X] Nice job including that dba directory with all the sql - good to have it backed up and easy to reproduce. Also, db design looks solid. 
- [X] Did you remember to open up port 8080 on AWS to public? I wasn't able to access the deployed app.
- [X] Take a look at .gitignore (there are examples in each week's exercise). Consider including one to prevent project workspace files, logs and the target directory from getting checked into version control.
- [x] Good work with the daos and tests. How are you keeping the db clean when testing? Any thoughts about using a database specifically for unit testing?

**Comment by ocollins** [comment]
Do I have to have a separate DB for testing? I do not think I really need it, but will if it is part of requirements.

**Comment by pawaitemadisoncollege** [comment]
You are right, it's not required, but does make testing and clean up after testing easier. It's up to you if you use a test db or not.
